common/main/eo.xml

```diff
@@ -861,7 +861,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<dateFormats>
 					<dateFormatLength type="full">
 						<dateFormat>
-							<pattern>EEEE, d-'a' 'de' MMMM y G</pattern>
+							<pattern draft="unconfirmed">EEEE, 'la' d-'a' 'de' MMMM y G</pattern>
```

Comment:
@macchiati is this going to damage the coverage level though? 

```suggestion
							<pattern draft="contributed">EEEE, 'la' d-'a' 'de' MMMM y G</pattern>
```

Reply:
Should all of the changes in this PR be "contributed" instead of "unconfirmed"?

Reply:
that's kind of my thought.

Reply:
or actually move the entire file to contributed